### PR TITLE
Make getProperties method public in BaseEntity

### PR DESCRIPTION
**Pull request by omrihq**
Addresses issue: https://github.com/googleapis/google-cloud-java/issues/5451

### CODE REVIEW

**Comment by omrihq** [comment]
I signed it!

**Comment by omrihq** [comment]
@igorbernstein2 Hi, I'm struggling to see the build failure here for formatting, mind pointing me in the right direction?

**Comment by igorbernstein2** [comment]
Please run
`mvn com.coveo:fmt-maven-plugin:format`

**Comment by omrihq** [comment]
Thanks @igorbernstein2, formatted and updated

**Comment by omrihq** [comment]
Hi @sduskis, seems like all checks have passed, is there someone in specific I request a review from?

File: google-cloud-clients/google-cloud-datastore/src/main/java/com/google/cloud/datastore/BaseEntity.java [hunk 1]
```diff
@@ -631,7 +631,8 @@ public Blob getBlob(String name) {
     return properties.keySet();
   }
 
-  ImmutableSortedMap<String, Value<?>> getProperties() {
+  /** Returns the properties. */
+  public ImmutableSortedMap<String, Value<?>> getProperties() {
```
Comment (chingor13) [inline google-cloud-clients/google-cloud-datastore/src/main/java/com/google/cloud/datastore/BaseEntity.java]: We don't want to expose Guava types as part of the public surface area. We may be able to investigate exposing this as a simple Map type.

**Pull request merged**